toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -487,11 +490,29 @@ func PopulateInstallRoot(installChroot *safechroot.Chroot, packagesToInstall []s
 		}
 	}
 
+	// Generate list of installed packages
+	GenerateContainerManifests(installChroot)
+
 	// Run post-install scripts from within the installroot chroot
 	err = runPostInstallScripts(installChroot, config)
 	return
 }
 
+func GenerateContainerManifests(installChroot *safechroot.Chroot) {
+	installRoot := filepath.Join(rootMountPoint, installChroot.RootDir())
+	rpmDir := filepath.Join(installRoot, rpmDependenciesDirectory)
+	rpmManifestDir := filepath.Join(installRoot, rpmManifestDirectory)
+	manifest1Path := filepath.Join(rpmManifestDir, "container-manifest-1")
+	manifest2Path := filepath.Join(rpmManifestDir, "container-manifest-2")
```

Comment:
Let's use more descriptive names for these files. One suggestion:

```suggestion
	manifest1Path := filepath.Join(rpmManifestDir, "container-manifest-all")
	manifest2Path := filepath.Join(rpmManifestDir, "container-manifest-all-detailed")
```

Reply:
sure, updated

Reply:
I've reverted back to the original filenames as requested by our partners